crypto/src/signature/mod.rs

```diff
@@ -63,47 +58,35 @@ impl Signature {
     ///
```

Comment:
I also think you can remove `ffi_impl_opaque` from `Signature`. I can't comment at the correct place